src/indigo/less/kbc-user.less

```diff
@@ -189,17 +143,6 @@
       float: left;
       margin: 9px .75em 0 0;
     }
-    .kbc-user-menu {
```

Comment:
@natocTo toto musi ostat

Reply:
To jsme nenašel v kbc-ui, to je v connection?

Reply:
jj, cast v connection ale tiez v kbc-ui-notifications https://github.com/search?q=org%3Akeboola+kbc-user-menu&type=Code